models/concept.go

```diff
@@ -69,14 +72,28 @@ func getNrPersonsWithData(conceptId int64, conceptsAndPersonsWithData []*Concept
 	return 0
 }
 
+// Retrieve just a simple concept info for a given conceptId.
+func (h Concept) RetrieveInfoBySourceIdAndConceptId(sourceId int, conceptId int64) (*ConceptSimple, error) {
+	conceptIds := make([]int64, 1)
+	conceptIds[0] = conceptId
+	result, err := h.RetrieveInfoBySourceIdAndConceptIds(sourceId, conceptIds)
+	if err != nil {
+		return nil, err
+	} else if len(result) == 0 {
+		// given concept_id not found, just return nil:
+		return nil, nil
```

Comment:
should we be returning nil for ConceptSimple when there is an error and also when there is not an error. Maybe we should return empty list when len(result) == 0?

Reply:
I think nil is fine here. It will just mean that the concept is not in DB. It is just delegating the responsibility of handling this to the calling function. We could also throw an error...but that is less generic. The calling function might want to use this function just to see if something exists in DB for example.